crates/bevy_sprite/src/render/mod.rs

```diff
@@ -528,13 +554,46 @@ pub fn queue_sprites(
 }
 
 #[allow(clippy::too_many_arguments)]
-pub fn prepare_sprites(
```

Comment:
This diff is pretty gnarly but it looks like it's just a (unrelated?) refactor of this system into two, right?

Reply:
It's not unrelated. These systems are split into two because one of them does per-sprite setup, `prepare_sprite_image_bind_groups`, and the other one does per-view setup, `prepare_sprite_view_bind_groups`.

Before this PR, it was done in one system because earlier view bind groups were always the same, even for different views. Now, since different views (cameras) can have different tonemapping settings, we need to be able to provide different LUTs for different views.
It's pretty close to what is currently done for
* 3d meshes: https://github.com/bevyengine/bevy/blob/main/crates/bevy_render/src/view/mod.rs#L788
* 2d meshes: https://github.com/bevyengine/bevy/blob/main/crates/bevy_sprite/src/mesh2d/mesh.rs#L587